PerformanceEfficiency/AppServiceAutoscalingSample/Readme.md

```diff
@@ -16,7 +16,7 @@ This margin between the scale-out and in and the threshold is recommended, consi
 
 Assuming there is 1 instance to start with. If the average CPU% across instances is 81 (the CPU% usage of the only instance), autoscale scales out adding a second instance. Then over time the CPU% falls to 60. Autoscale's scale-in rule estimates the final state if it were to scale-in. For example, 60 x 2 (current instance count) = 120 / 1 (final number of instances when scaled down) = 120. So autoscale does not scale-in because it would have to scale-out again immediately. Instead, it skips scaling down. The next time autoscale checks, the CPU continues to fall to 30%. It estimates again - 30 x 2 instance = 60 / 1 instance = 60, which is below the scale-out threshold of 80, so it scales in successfully to 1 instance.
 
-The duration is set to 5 minutes. This is amount of time that Autoscale engine will look back for metrics. So in this case, 5 minutes means that every time autoscale runs, it will query metrics for the past 5 minutes. This allows your metrics to stabilize and avoids reacting to transient spikes. 
+The duration is set to 5 minutes. This is the amount of time that the Autoscale engine will look back for metrics. So in this case, 5 minutes means that every time autoscale runs, it will query metrics for the past 5 minutes. This allows your metrics to stabilize and avoids reacting to transient spikes. 
```

Comment:
```suggestion
The amount of time that the autoscaler checks for metrics is determined by the the duration, which is set to 5 minutes.  So, every time autoscale runs, it will query metrics for 5 minutes. This way the autoscaler can get stable metrics instead of reacting to transient spikes. 
```